Assets/MRTK/SDK/Features/Utilities/Migration/Tools/MigrationTool.cs

```diff
@@ -362,6 +400,23 @@ private static bool IsSceneAsset(Object selectedObject)
         {
             return selectedObject is SceneAsset;
         }
+
+        /// <summary>
+        /// Utility class to keep migration status of each object
+        /// </summary>
+        public class MigrationStatus
+        {
+            public bool isProcessed;
```

Comment:
Especially since this is a `public` class, we should be sure to follow [our encapsulation recommendations](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation). It looks to me like these can just be turned into auto-properties.

Reply:
Also some `<summary>` tags.